express-api/src/controllers/agencies/agenciesController.ts

```diff
@@ -21,12 +21,12 @@ export const getAgencies = async (req: Request, res: Response) => {
             "bearerAuth": []
       }]
    */
-  const kcUser = req.user as KeycloakUser;
+  const ssoUser = req.user as SSOUser;
```

Comment:
I think req.user should already be of type SSOUser

Reply:
updated